apps/codeforafrica/src/components/ImpactCard/ImpactCard.js

```diff
@@ -0,0 +1,88 @@
+import { Card, CardActionArea, Typography, Box } from "@mui/material";
+import { styled } from "@mui/material/styles";
+import Image from "next/image";
+import PropTypes from "prop-types";
+import React from "react";
+
+const ImpactCardRoot = styled(Card, {
+  slot: "Root",
+})(() => ({}));
+
+const ImpactCard = React.forwardRef(function ImpactCard(props, ref) {
+  const { image, title, number, description } = props;
+
+  if (!image && !title) {
+    return null;
+  }
+
+  return (
+    <ImpactCardRoot ref={ref}>
+      <CardActionArea>
+        <Box
+          sx={{
+            display: "flex",
+            fontWeight: 700,
+            borderBottom: "1px solid #1020E1",
+            paddingBottom: "0.75rem",
+          }}
+        >
+          <Image src={image.url} alt={image.alt} width={30} height={30} />
+          <Typography
+            variant="display3"
+            sx={{
+              display: "inline",
+              color: "primary.main",
+              marginLeft: "0.93rem",
+            }}
+          >
+            {title}
+          </Typography>
+        </Box>
+        {number && (
+          <Typography
+            variant="display1"
+            sx={{
+              color: "primary.main",
+              fontWeight: 300,
```

Comment:
1. Use RichTypography component which checks for null
2. Use correct `variant` so you don't need to specify `fontWeight` (add one if it's missing).